zulipterminal/ui_tools/views.py

```diff
@@ -1988,14 +1988,27 @@ def add_or_remove_selected_emoji(self, emoji_code: str, emoji_name: str) -> None
         else:
             self.selected_emojis.update({emoji_code: emoji_name})
 
+    def count_reactions(self, emoji_code: str) -> int:
+        num_reacts = 0
+        for reaction in self.message["reactions"]:
+            if reaction["emoji_code"] == emoji_code:
+                num_reacts += 1
+        return num_reacts
+
     def generate_emoji_buttons(
         self, emoji_units: List[Tuple[str, str, List[str]]]
     ) -> List[EmojiButton]:
+        emoji_units = sorted(
+            emoji_units,
+            key=lambda emoji_unit: self.count_reactions(emoji_unit[1]),
```

Comment:
This only explicitly sorts by count here?

To avoid calling count_reactions multiple times, we could sort the EmojiButton list instead?